### DISCUSSION TITLE
Is there a way to create JWT like with jsonwebtoken

### DISCUSSION THREAD

**Discussion by azimari-toure-ikbal** [honojs/hono#1468] [Q&A] [answered]
Hello ! I want to know if there's a way to create jwt tokens using hono built-in functions ? I saw there's something like c.set("jwtPayload", whateverValues) but will it work ?
I want to build something like : 
`  const tempToken = jwt.sign(
    { iss: "my-app", sub: userId },
    process.env.SECRET!,
    { expiresIn: DURATIOM }
  );`

**Comment by jaredLunde** [comment, marked as answer]
Hono has a JWT util. As of 3.7.0, this can be used like:

```ts
import * as Jwt from 'hono/jwt'

const token = await Jwt.sign(
    {
      sub: "SESSION_ID",
      jti: "TOKEN_ID",
      iat: Math.floor(Date.now() / 1000),
      exp: Math.floor(Date.now() / 1000) + expiresIn,
      iss: "ISSUER",
      aud: "AUDIENCE",
    },
    env.JWT_SECRET,
  );
```

**Reply by azimari-toure-ikbal** [reply]
This is actually fire ! How can you make such a magical experience building server ?! Almost no dependencies needed, stuff are just here out of the box. Man i'm on fire for Hono (get it ?)

**Reply by ngdbao** [reply]
I see We cannot include 'jwt header' when signing JWT ? or manually "KID" included in header
